apps/docs/content/guides/auth/auth-anonymous.mdx

```diff
@@ -80,6 +80,15 @@ Call the [`signInAnonymously()`](/docs/reference/kotlin/auth-signinanonymously)
 supabase.auth.signInAnonymously()
 ```
 
+</TabPanel>
+<TabPanel id="python" label="Python">
+
+Call the [`sign_in_anonymously()`](/docs/reference/python/auth-signinanonymously) method:
+
+```python
+response = await supabase.auth.sign_in_anonymously()
```

Comment:
It might be worth removing the `await` from all of the `python` examples as most Python frameworks are sync.